### DISCUSSION TITLE
English GUI?

### DISCUSSION THREAD

**Discussion by roamindev** [PriesiaMioShirakana/DragonianVoice#28] [Q&A] [answered]
First, great work on this project! I'm wondering if there is a way to switch languages or an english version of the GUI available? The UI is intuitive enough that I was able to get my model working for basic conversion, but I have no idea what the different settings/options are to try and tweak things for the best output. Thanks!

Edit: I was able to take screenshots of the GUI and run them through a translator and that helped a bit, but, I'm still curious about what each of the different settings actually does. Is there any info here or elsewhere that I could read up on to help me understand what the settings do?

**Comment by PriesiaMioShirakana** [comment, marked as answer]
Other languages will be supported in the future
